sky/engine/public/platform/sky_settings.h

```diff
@@ -17,6 +17,7 @@ struct SkySettings {
   bool start_paused = false;
   bool enable_dart_checked_mode = false;
   bool trace_startup = false;
+  bool complete_timeline = false;
```

Comment:
By default the VM timeline uses a ring buffer and old events are dropped.

--complete_timeline is short hand for:

--timeline_streams=all
--timeline_recorder=endless

All timeline categories are enabled and we use the 'endless' recorder which will keep allocating memory for timeline events until malloc fails.

--startup_timeline is short hand for:

--timeline_streams=all
--timeline_recorder=startup

All timeline categories are enabled and we use the 'startup' recorder which will record up to 8k events and then start dropping events.